Cargo.toml

```
@@ -1744,7 +1744,7 @@ doc-scrape-examples = true
 
 [package.metadata.example.breakout]
 name = "Breakout"
-description = "An implementation of the classic game \"Breakout\""
+description = "An implementation of the classic game \"Breakout\". Demonstrates Bevy's stepping capabilities if compiled with the `bevy_debug_stepping` feature."
```

```suggestion
description = "An implementation of the classic game \"Breakout\"."
```
This is a plus, but not the first goal of the example. I think the comment on top of the example should be enough